### PR TITLE
add and test for more versionless versions

### PR DESCRIPTION
**Pull request by rsherget**

### CODE REVIEW

**Comment by rsherget** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_fh4isBeIEe-aC7a28EbkCw

**Comment by jhanders34** [comment]
#libby #build

**Comment by jhanders34** [comment]
#libby #build

**Comment by rsherget** [comment]
#build

File: dev/com.ibm.ws.kernel.feature.resolver_fat/fat/src/com/ibm/ws/feature/tests/PlatformConstants.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                    "MicroProfile-4.0, MicroProfile-3.3, MicroProfile-3.2, MicroProfile-3.0, MicroProfile-2.2, MicroProfile-2.1, " +
+                    "MicroProfile-2.0, MicroProfile-1.4, MicroProfile-1.3, MicroProfile-1.2, MicroProfile-1.0";
+
+    public static final String MICROPROFILE_ASCENDING = "MicroProfile-1.0, MicroProfile-1.2, MicroProfile-1.3, MicroProfile-1.4, " +
+                    "MicroProfile-2.0, MicroProfile-2.1, MicroProfile-2.2, MicroProfile-3.0, MicroProfile-3.2, MicroProfile-3.3, " +
+                    "MicroProfile-4.0, MicroProfile-4.1, MicroProfile-5.0, MicroProfile-6.0, MicroProfile-6.1";
+
+    public static final String JAKARTA_DESCENDING = "jakartaee-11.0, jakartaee-10.0, jakartaee-9.0, jakartaee-8.0, jakartaee-7.0, jakartaee-6.0";
+
+    public static final String JAKARTA_ASCENDIN = "jakartaee-6.0, jakartaee-7.0, jakartaee-8.0, jakartaee-9.0, jakartaee-10.0, jakartaee-11.0";
```
Comment (jhanders34) [inline dev/com.ibm.ws.kernel.feature.resolver_fat/fat/src/com/ibm/ws/feature/tests/PlatformConstants.java]: Should `jakartaee-6.0` be `javaee-6.0` and similar for `7.0` and `8.0`?  Also `ASCENDIN` should be `ASCENDING`

Comment (rsherget) [reply dev/com.ibm.ws.kernel.feature.resolver_fat/fat/src/com/ibm/ws/feature/tests/PlatformConstants.java]: Yes, yes they should. They should be fixed now

**Comment by rsherget** [comment]
#build

**Review by jhanders34 — APPROVED** [review]

**Comment by jhanders34** [comment]
#libby

**Review by cbridgha — APPROVED** [review]
Looks good thanks

**Comment by rsherget** [comment]
RTC build's only error was unrelated to change, merging

**Pull request merged**